#
# Definition of all trackable objects
# Identifier corresponds to Trackable ID set in Tracking Tools
#
rigid_bodies:
    '1':
        pose: qrotor1/pose
        pose2d: qrotor1/ground_pose
        child_frame_id: qrotor1/base_link
        parent_frame_id: world
        transform: /qrotor1
    '10':
        pose: load1/pose
        pose2d: load1/ground_pose
        child_frame_id: load1/base_link
        parent_frame_id: world
        transform: /load1


optitrack_config:
        multicast_address: 239.255.42.99
